cocotb/share/makefiles/simulators/Makefile.aldec

```diff
@@ -84,17 +84,17 @@ endif
 
 GPI_EXTRA:=
 ifeq ($(TOPLEVEL_LANG),verilog)
-    GPI_ARGS = -pli $(LIB_DIR)/libcocotbvpi
+    GPI_ARGS = -pli $(LIB_DIR)/aldec/libcocotbvpi
 ifneq ($(VHDL_SOURCES),)
     GPI_EXTRA = cocotbvhpi
 endif
 
 else ifeq ($(TOPLEVEL_LANG),vhdl)
 
 ifeq ($(OS),Msys)
-    VHPI_LIB = $(shell sh -c 'cd $(LIB_DIR) && pwd -W')/libcocotbvhpi
+    VHPI_LIB = $(shell sh -c 'cd $(LIB_DIR) && pwd -W')/aldec/libcocotbvhpi
 else
-    VHPI_LIB = $(LIB_DIR)/libcocotbvhpi
+    VHPI_LIB = $(LIB_DIR)/aldec/libcocotbvhpi
```

Comment:
Are we at a point now where we can make this `libcocotbvhpi.aldec` or similar, which eliminates some of the rpath stuff?

Reply:
No. https://github.com/cocotb/cocotb/pull/1457. The change will be minimal and eventually may be needed to ger` simulator` module moved to the right `cocotb` directory (will do next).

Reply:
Ah, thought that was already merged. Do you want to get that in first, or get this in first?

Reply:
If you help. It would be great. I will try to fix what is left #1457 today.